### PR TITLE
docs(swipe): modify custom indicator example

### PR DESCRIPTION
**Pull request by chouchouji**
## Checklist

List of tasks you have already done and plan to do.

- [x] modify custom indicator example

## Change information

Describe your modifications here.

## Issues

The issues you want to close, formatted as close #1.

## Related Links

Links related to this pr.

### CODE REVIEW

File: packages/varlet-ui/src/swipe/docs/en-US.md [hunk 1]
```diff
@@ -174,11 +180,9 @@ import { Snackbar } from '@varlet/ui'
     <var-swipe-item>
       <img class="swipe-example-image" src="https://varlet.gitee.io/varlet-ui/cat3.jpg">
     </var-swipe-item>
-
     <template #indicator="{ index, length }">
-      <div class="swipe-example-indicators">
-        {{ index + 1 }} / {{ length }}
-      </div>
+      <div class="swipe-example-indicators-left" :class="index === 0 ? 'swipe-example-active-indicator' : 'swipe-example-inactive-indicator'" @click="index !== 0 && swipeRef.prev()"><var-icon name="chevron-left" /></div>
+      <div class="swipe-example-indicators-right" :class="index === length - 1 ? 'swipe-example-active-indicator' : 'swipe-example-inactive-indicator'" @click="index !== length - 1 && swipeRef.next()"><var-icon name="chevron-right" /></div>
```
Comment (haoziqaq) [inline packages/varlet-ui/src/swipe/docs/en-US.md:185]: The inline logic is too complicated, it is recommended to extract it into a function.

Comment (chouchouji) [reply packages/varlet-ui/src/swipe/docs/en-US.md]: ok, I will delete it.

File: packages/varlet-ui/src/swipe/docs/en-US.md [hunk 2]
```diff
@@ -186,25 +197,37 @@ import { Snackbar } from '@varlet/ui'
 <style>
 .swipe-example {
   height: 160px;
+  position: relative;
```
[the remark this answers is not in the document]
Comment (chouchouji) [reply packages/varlet-ui/src/swipe/docs/en-US.md:200]: ok, I will delete it.

[the remark this answers is not in the document]
Comment (chouchouji) [reply packages/varlet-ui/src/swipe/docs/en-US.md:200]: done!

**Comment by chouchouji** [comment]
@haoziqaq sorry, I failed to push my code(delete position), but i didn't find it. I has pushed successfully.

**Pull request merged**